fp.js

```diff
@@ -23,6 +35,11 @@ people.forEach(person => {
     peopleYoungerThan35.push(person)
   }
 })
+function youngPeople (value,index,array){
```

Comment:
You don't need these arguments. Filter should look something like this, where the return value is whatever condition you want the items in the new array to meet. In this case, each person should be under the age of 35.

```js
const peopleUnder35 = people.filter(person => {
  // this makes it so that the only people end up in the new array are under 35
  return person.age < 35;
})
```